src/DynamicMvvm.FluentValidation/DynamicMvvm.FluentValidation.csproj

```diff
@@ -7,15 +7,26 @@
 		<Company>nventive</Company>
 		<AssemblyName>Chinook.DynamicMvvm.FluentValidation</AssemblyName>
 		<PackageId>Chinook.DynamicMvvm.FluentValidation</PackageId>
-		<Description>Chinook.DynamicMvvm.FluentValidation</Description>
+		<Description>Chinook.DynamicMvvm is a collection of extensible MVVM libraries for declarative ViewModels.</Description>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageReadmeFile>README.md</PackageReadmeFile>
+		<PackageTags>mvvm;xamarin;ios;android;uwp;chinook</PackageTags>
```

Comment:
Do we want to add .NET too ? :)

Reply:
I thought about it, but nuget packages are all for .NET 😅

Reply:
Oh... yes obviously lol